### PR TITLE
AC Stub Test - LRAC-8299 | master

### PR DESCRIPTION
**Pull request by rafaellajordao**
https://issues.liferay.com/browse/LRAC-8299

### CODE REVIEW

**Comment by ViniLopes20** [comment]
Hi, @rafaellajordao. Commits always starting with LRAC-XXXX (XXX -> Your branch number).

because it won't give an error when calling `ci:forward`

**Comment by alexcandidos** [comment]
Also @rafaellajordao , please add the ticket ID in front of the PR message :)

**Comment by rafaellajordao** [comment]
Fixed! @alexcandidos and @ViniLopes20!

**Comment by ViniLopes20** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase [hunk 1]
```diff
@@ -31,6 +31,69 @@ definition {
 		ACUtils.tearDownAC();
 	}
 
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8293 | Test Summary: Assert 180 Day time period filter in Sites"
+	@ignore = "true"
+	@priority = "5"
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase:36]: Hi @rafaellajordao , this tc is priority 3 on the test map.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase [hunk 2]
```diff
@@ -31,6 +31,69 @@ definition {
...
+	test Assert180DayTimePeriodFilterInSites {
+
+		// TODO LRAC-8293 Assert180DayTimePeriodFilterInSites
+
+	}
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8291 | Test Summary: Assert Sites Activities 180 Days Time Period Filter In Page Table"
+	@ignore = "true"
+	@priority = "5"
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase:45]: Hi @rafaellajordao , this tc is priority 3 on the test map.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase [hunk 3]
```diff
@@ -31,6 +31,69 @@ definition {
...
+	test AssertSitesActivities180DaysTimePeriodFilterInPageTable {
+
+		// TODO LRAC-8291 AssertSitesActivities180DaysTimePeriodFilterInPageTable
+
+	}
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8282 | Test Summary: Anonymous converts to known"
+	@ignore = "true"
+	@priority = "5"
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase:54]: Hi @rafaellajordao , this tc is priority 4 on the test map.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase [hunk 4]
```diff
@@ -40,4 +103,85 @@ definition {
...
+	test CanSitesActivityTimeFilterUseCustomRange {
+
+		// TODO LRAC-8295 CanSitesActivityTimeFilterUseCustomRange
+
+	}
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8290 | Test Summary: Unique Visitors label is shown on the tabs of Pages dashboards"
+	@ignore = "true"
+	@priority = "5"
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase:135]: Hi @rafaellajordao , this tc is priority 3 on the test map.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase [hunk 5]
```diff
@@ -40,4 +103,85 @@ definition {
...
+	test CanUniqueVisitorsLabelShowPagesDashboards {
+
+		// TODO LRAC-8290 CanUniqueVisitorsLabelShowSitesDashboards
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase:138]: hi @rafaellajordao it should be this // TODO LRAC-8290 CanUniqueVisitorsLabelShowPagesDashboards

**Review by alexcandidos — CHANGES_REQUESTED** [review]
hi @rafaellajordao , please see my comments and update the priority of some tests.

Thanks!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase [hunk 6]
```diff
@@ -31,6 +31,69 @@ definition {
...
+	}
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8282 | Test Summary: Anonymous converts to known"
+	@ignore = "true"
+	@priority = "3"
+	test CanAnonymousIndividualsConvertsToKnownIndividuals {
+
+		// TODO LRAC-8282 CanAnonymousIndividualsConvertsToKnownIndividuals
+
+	}
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase:59]: hi @rafaellajordao , this tc is a priority 4 in our test map.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase [hunk 7]
```diff
@@ -31,6 +31,69 @@ definition {
...
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8275 | Test Summary: Site Activity Card: Can filter graph by day"
+	@ignore = "true"
+	@priority = "3"
+	test CanFilterActivityCardGraphByDay {
+
+		// TODO LRAC-8275 CanFilterActivityCardGraphByDay
+
+	}
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase:68]: hi @rafaellajordao , this tc is a priority 5 in our test map.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase [hunk 8]
```diff
@@ -31,6 +31,69 @@ definition {
...
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8278 | Test Summary: Site Activity Card: Can filter graph by month"
+	@ignore = "true"
+	@priority = "3"
+	test CanFilterActivityCardGraphByMonth {
+
+		// TODO LRAC-8278 CanFilterActivityCardGraphByMonth
+
+	}
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase:77]: hi @rafaellajordao , this tc is a priority 5 in our test map.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase [hunk 9]
```diff
@@ -31,6 +31,69 @@ definition {
...
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8276 | Test Summary: Site Activity Card: Can filter graph by week"
+	@ignore = "true"
+	@priority = "3"
+	test CanFilterActivityCardGraphByWeek {
+
+		// TODO LRAC-8276 CanFilterActivityCardGraphByWeek
+
+	}
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase:86]: hi @rafaellajordao , this tc is a priority 5 in our test map.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase [hunk 10]
```diff
@@ -31,6 +31,69 @@ definition {
...
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8279 | Test Summary: Site Activity Card: Can hover over a point on the graph to get detailed stats"
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase:88]: hi @rafaellajordao , this tc is a priority 5 in our test map.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase [hunk 11]
```diff
@@ -40,4 +103,85 @@ definition {
...
 	}
 
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8280 | Test Summary: Anon and known visitors legend shown"
+	@ignore = "true"
+	@priority = "3"
+	test CanShownAnonAndKnownVisitorsLegend {
+
+		// TODO LRAC-8280 CanShownAnonAndKnownVisitorsLegend
+
+	}
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase:113]: hi @rafaellajordao , this tc is a priority 5 in our test map.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase [hunk 12]
```diff
@@ -40,4 +103,85 @@ definition {
...
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8281 | Test Summary: Anon and known visitors popover shows breakdown"
+	@ignore = "true"
+	@priority = "3"
+	test CanShownAnonAndKnownVisitorsPopoverBreakdown {
+
+		// TODO LRAC-8281 CanShownAnonAndKnownVisitorsPopoverBreakdown
+
+	}
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase:122]: hi @rafaellajordao  this tc is a priority 5 in our test map.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase [hunk 13]
```diff
@@ -40,4 +103,85 @@ definition {
...
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8295 | Test Summary: Assert Sites Activity time filter able to use Custom Range"
+	@ignore = "true"
+	@priority = "3"
+	test CanSitesActivityTimeFilterUseCustomRange {
+
+		// TODO LRAC-8295 CanSitesActivityTimeFilterUseCustomRange
+
+	}
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase:131]: hi @rafaellajordao , this tc is a priority 5 in our test map.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase [hunk 14]
```diff
@@ -40,4 +103,85 @@ definition {
...
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8288 | Test Summary: Unique Visitors label is shown on the tabs of Sites dashboards"
+	@ignore = "true"
+	@priority = "5"
+	test CanUniqueVisitorsLabelShowSitesDashboards {
+
+		// TODO LRAC-8288 CanUniqueVisitorsLabelShowSitesDashboards
+
+	}
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase:149]: hi @rafaellajordao , this tc is a priority 3 in our test map.

**Review by alexcandidos — CHANGES_REQUESTED** [review]
hi @rafaellajordao , please take a look at my comments, some priorities are wrong.

Thanks!

**Comment by alexcandidos** [comment]
Hi @rafaellajordao , this test `CanSitesActivityTimeFilterUseCustomRange` is still with the wrong priority, it should be 5 and not 3.

Could you please fix it? Thanks!

**Comment by alexcandidos** [comment]
Hi @rafaellajordao , Thanks for working on it!

The code looks good to me!

**Review by alexcandidos — APPROVED** [review]
Hi @rafaellajordao , Thanks for working on it!

The code looks good to me!

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**